packages/databases-collections/src/components/zero-graphic.tsx

```diff
@@ -36,4 +36,38 @@ const ZeroGraphic = () => {
   );
 };
 
-export { ZeroGraphic };
+function AddDataZeroGraphic(
```

Comment:
NIT: usual mega silly comment about naming, but can we use external language also here, and say "load sample data" instead of "add data"?

Reply:
Adjusted the name and moved everything load data related to a new file